### ISSUE TITLE
Landing page [id] attributes are not unique

### ISSUE THREAD

**Issue by rmjordas** [projectunic0rn/mentorship-ui#314]
> The value of an id attribute must be unique to prevent other instances from being overlooked by assistive technologies. [Learn more](https://dequeuniversity.com/rules/axe/2.2/duplicate-id?application=lighthouse).

Refer to the [Lighthouse report](https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://www.mentrship.com/) for the list of failing elements.
